### ISSUE TITLE
Support for .schema files in addition to .schematic

### ISSUE THREAD

**Issue by MarkC0x** [TheBrunoRM/Skywars#5]
I know this is probably asking a lot 🙂  but it would be amazing if this plugin could be enhanced to support the newer `.schem` format.  Creating `.schematic` files appears to be getting harder as it's such an old format now, which means that leveraging any arenas you create yourself seems to be very difficult (unless I am missing something - which is entirely possible).

When I attempt (long-shot) to use a `schem` file created from the latest version of WorldEdit (`7.2.10`) an enormous amount of `IOException` stack traces get printed:

```
[Server thread/WARN]: java.io.IOException: Invalid tag type: 11.
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTagPayload(NBTInputStream.java:170)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTag(NBTInputStream.java:102)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTagPayload(NBTInputStream.java:160)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTagPayload(NBTInputStream.java:149)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTag(NBTInputStream.java:102)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTagPayload(NBTInputStream.java:160)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTag(NBTInputStream.java:102)
[Server thread/WARN]:         at org.jnbt.NBTInputStream.readTag(NBTInputStream.java:80)
[Server thread/WARN]:         at me.brunorm.skywars.schematics.SchematicHandler.loadSchematic(SchematicHandler.java:356)
[Server thread/WARN]:         at me.brunorm.skywars.structures.SkywarsMap.loadSchematic(SkywarsMap.java:258)
[Server thread/WARN]:         at me.brunorm.skywars.structures.SkywarsMap.getSchematic(SkywarsMap.java:265)
[Server thread/WARN]:         at me.brunorm.skywars.structures.Arena.isInBoundaries(Arena.java:593)
[Server thread/WARN]:         at me.brunorm.skywars.structures.Arena.isInBoundaries(Arena.java:589)
[Server thread/WARN]:         at me.brunorm.skywars.events.Events.onMove(Events.java:49)
...
```

If this is too big an enhancement request then I totally get it, but I I thought I would ask (even though you mention this format incompatibility on your main GitHub page for this plugin).

**Comment by TheBrunoRM** [comment]
I already started working on making schem files work, but I haven't done anything that works yet. At some point, I will make it read schem files, but for now, I have some problems that I need to get rid of in order to be able to do this.

**Comment by TheBrunoRM** [comment]
I discovered the `Invalid tag type: 11` error is because the old version of [JNBT](https://github.com/Morlok8k/JNBT) that I was using doesn't have the tag type `IntArray`. I will start using another library that allows me to read both schem and schematic files. I'm currently testing and I may come up with a solution where the plugin can use both file types.

**Comment by TheBrunoRM** [comment]
Fixed in 035a5355ac2046adfb23cc2eb3e5509ad6b44e9c
